File: 1_easy/00026_remove_duplicates_from_sorted_array/solution.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:

        nums_len = len(nums)
        index = 0

        for i in range(1, nums_len):
            if nums[i] != nums[i - 1]:
                index += 1
                nums[index] = nums[i]

        return index + 1

File: 1_easy/00026_remove_duplicates_from_sorted_array/solution.rs
impl Solution {
    pub fn remove_duplicates(nums: &mut Vec<i32>) -> i32 {

        let nums_len = nums.len();
        let mut index = 0_usize;

        for i in 1..nums_len {
            if nums[i] != nums[i - 1] {
                index += 1;
                nums[index] = nums[i];
            }
        }

        index as i32 + 1
    }
}
